ward/collect.py

```diff
@@ -115,21 +126,39 @@ def get_tests_in_modules(
                     marker=meta.marker,
                     description=meta.description or "",
                     capture_output=capture_output,
+                    tags=meta.tags or [],
                 )
 
 
 def search_generally(
-    tests: Iterable[Test], query: str = ""
+    tests: Iterable[Test],
+    query: str = "",
+    tags: Optional[Expression] = None,
+    exclude_tags: Optional[Expression] = None,
```

Comment:
What's the use-case for `exclude_tags`? Can't you just use `not` in your query?

Reply:
Indeed. I'll remove that the next time I work on it.